CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add `activeSpeakerPolicy` and `videoUplinkBandwidthPolicy` to `MeetingProvider` props.
 
 ### Changed
+- Add `reconnectTimeoutMs` as an optional parameter `MeetingManagerConfig` to control the configured timeout before starting a reconnect.
```

Comment:
Have to update the changelog to match the corrected variable description.

Reply:
Thats a great observation.